src/main/java/com/igot/cb/transactional/service/RequestHandlerServiceImpl.java

```diff
@@ -0,0 +1,69 @@
+package com.igot.cb.transactional.service;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.igot.cb.demand.service.DemandServiceImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Service;
+import org.springframework.util.CollectionUtils;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.client.RestTemplate;
+
+import java.util.HashMap;
+import java.util.Map;
+@Service
+public class RequestHandlerServiceImpl {
+    private Logger log = LoggerFactory.getLogger(DemandServiceImpl.class);
+
+    @Autowired
+    private RestTemplate restTemplate;
+
+    public Map<String, Object> fetchResultUsingPost(String uri, Object request, Map<String, String> headersValues) {
```

Comment:
Don't we have already once service to send out request from this one?

Reply:
no